x/oracle/types/errors.go

```diff
@@ -7,4 +7,6 @@ var (
 	ErrGetOracleRegistration = sdkerrors.Register(ModuleName, 2, "error while getting a oracle registration")
 	ErrGetOracle             = sdkerrors.Register(ModuleName, 3, "error while getting a oracle")
 	ErrOracleNotFound        = sdkerrors.Register(ModuleName, 4, "oracle not found")
+	ErrInvalidUniqueId            = sdkerrors.Register(ModuleName, 5, "invalid unique id")
```

Comment:
```suggestion
	ErrInvalidUniqueID            = sdkerrors.Register(ModuleName, 5, "invalid unique id")
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms